src/containers/signupForm/ducks/types.tsx

```diff
@@ -0,0 +1,78 @@
+import { ThunkAction } from 'redux-thunk';
+import { ProtectedApiExtraArgs } from '../../../api/protectedApiClient';
+import { C4CState } from '../../../store';
+import { SetContactsActions } from '../../setContacts/actions';
+
+export enum SignupState {
+  SignupDirectory,
+  GeneralMemberFormOne,
+  GeneralMemberVerification,
+  GeneralMemberConfirmation,
```

Comment:
Shouldn't these be something like `PrimarySignupForm`, `SetContacts`, `SubmitPFRequest`?